packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx

```diff
@@ -121,7 +120,9 @@ function CategoryList({
                   embedded,
                   style: {
                     ...(showHiddenItems &&
-                      item.hidden && { color: theme.pageTextSubdued }),
+                      item.hidden && {
+                        color: theme.pageTextSubdued,
+                      }),
```

Comment:
Did this fix a lint issue? If not, I would revert.
```suggestion
                      item.hidden && { color: theme.pageTextSubdued }),
```